source/includes/fundamentals/code-examples/connection/AtlasConnection.cs

```diff
@@ -1,11 +1,18 @@
 using MongoDB.Driver;
+using MongoDB.Bson;
 
-// Connection URI
+// Replace the placeholders with your credentials
 const string connectionUri = "mongodb+srv://<username>:<password>@cluster0.sample.mongodb.net/?retryWrites=true&w=majority";
 
-var serverApi = new ServerApi(ServerApiVersion.V1);
 var settings = MongoClientSettings.FromConnectionString(connectionString);
+
+// Set the ServerApi field of the settings object to Stable API version 1
+var serverApi = new ServerApi(ServerApiVersion.V1);
```

Comment:
Suggestion:
I think you probably don't need to set this to a variable. I'm not able to think of any advantages in splitting it into two lines since it's fairly short and the variable isn't used anywhere else.

I'd recommend combining lines 10 and 11 unless there's a good reason not to. E.g.:

```
settings.ServerApi = new ServerApi(ServerApiVersion.V1);
```